docker-hub/api/latest.yaml

```diff
@@ -1,104 +1,137 @@
-swagger: '2.0'
-
-host: hub.docker.com
-basePath: '/'
-schemes:
-  - https
-consumes:
-  - application/json
-produces:
-  - application/json
+openapi: 3.0.0
 info:
   title: Docker HUB API
-  version: 'beta'
+  version: beta
   x-logo:
     url: https://docs.docker.com/images/logo-docker-main.png
     href: /reference
-  description: |
-    Docker Hub is a service provided by Docker for finding and sharing container images with your team.
+  description: >
+    Docker Hub is a service provided by Docker for finding and sharing container
+    images with your team.
+
     It is the world's largest library and community for container images.
+
     In addition to the [Docker Hub UI](https://docs.docker.com/docker-hub/) and [Docker Hub CLI tool](https://github.com/docker/hub-tool#readme) (currently experimental),
+
     Docker provides an API that allows you to interact with Docker Hub.
-    Browse through the Docker Hub API documentation to explore the supported endpoints.
 
+    Browse through the Docker Hub API documentation to explore the supported endpoints.
 tags:
   - name: resources
     x-displayName: Resources
-    description: |
+    description: >
       The following resources are available to interact with the documented API:
 
+
       - <a href="https://github.com/docker/hub-tool#readme" target="_blank">Docker Hub CLI tool</a> (currently experimental)
+
       - <a href="https://www.postman.com/devrel/workspace/docker-hub/collection/13191452-7d2f1375-9705-4db6-a47b-8235286e58a1" target="_blank">Postman Collection</a> (built by the Postman team)
```

Comment:
Not new, so probably best for a separate PR, but looks like this link is broken (no longer exists); https://www.postman.com/devrel/workspace/docker-hub/collection/13191452-7d2f1375-9705-4db6-a47b-8235286e58a1

Reply:
Yep, looks like we should raise that as a separate issue.